packages/docs/components/sidebar-logo.tsx

```diff
@@ -0,0 +1,31 @@
+"use client";
+
+import Image from "next/image";
+import { useState } from "react";
+
+interface SidebarLogoProps {
+  src: string;
+  alt: string;
+  width?: number;
+  height?: number;
+  className?: string;
+}
+
+export function SidebarLogo({ src, alt, width = 24, height = 24, className = "h-5" }: SidebarLogoProps) {
```

Comment:
Following the import suggestion, change with `ImageProps`
```suggestion
export function SidebarLogo({ src, alt, width = 24, height = 24, className = "h-5" }: ImageProps) {
```